remote/errors.go

```diff
@@ -56,14 +56,13 @@ func UnsupportedResourceKind(err error) error {
 		Type: fluxerr.User,
 		Help: err.Error() + `
 
-The version of the agent running in your cluster (fluxd) can only
-release updates to named deployments. The ability to release to other
-kinds of pod controllers (such as daemon sets and stateful sets) will
-be added in a future version of flux.
-
-Important - releasing by service name is no longer supported - if
-you're using an old version of fluxctl that does not let you specify
-a kind other than 'service', you will need to upgrade it.
+The version of the agent running in your cluster (fluxd) can only release
+updates to pod controllers such as Deployments, DaemonSets, StatefulSets and
+CronJobs.
+
+Releasing by service name is no longer supported - if you're using an old
+version of fluxctl that does not let you specify a kind other than 'service',
+you will need to upgrade it.
```

Comment:
Yeah, nice.

Reply:
It occurs to me that this could be generated from the value in resourcekinds.go. Gilding the lily perhaps ...